tutorials/tmva/TMVA_RNN_Classification.C

```diff
@@ -190,7 +190,7 @@ void TMVA_RNN_Classification(int use_type = 1)
    useKeras = false;
 #endif
 
-   int num_threads = 0;   // use by default all threads
+   int num_threads = 32;
```

Comment:
If I understand correctly this changes the number of threads spawned from NCORES to 32, which might be problematic on machines with less than 32 cores.

For RDF tests sometimes we use `std::min(4u, std::thread::hardware_concurrency())`, maybe something like that is better? ping @lmoneta

Reply:
It changes it to a maximum of 32. If there are less cores, fewer cores will be used.
But, yes, the OMP_NUM_THREADS env variable should be set to the number of threads actually allocated, not the number requested. PR updated.